NickvisionMoney.GNOME/Blueprints/group_dialog.blp

```diff
@@ -1,23 +1,57 @@
 using Gtk 4.0;
 using Adw 1;
 
-Adw.MessageDialog _root {
-  heading: _("Group");
+Adw.Window _root {
   default-width: 360;
-  hide-on-close: true;
   modal: true;
+  resizable: false;
 
-  Adw.PreferencesGroup {
-    Adw.EntryRow _nameRow {
-      title: _("Name.Field");
-      input-hints: spellcheck;
-      activates-default: true;
+  content: Gtk.Box {
+    orientation: vertical;
+    spacing: 0;
```

Comment:
Is this necessary?

Reply:
I didn't try to do it another way, but the docs say:
> Using [GtkWindow:titlebar](https://docs.gtk.org/gtk4/property.Window.titlebar.html) or [GtkWindow:child](https://docs.gtk.org/gtk4/property.Window.child.html) is not supported and will result in a crash. Use [AdwWindow:content](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/property.Window.content.html) instead.

And `content` should be one widget, so I use the box.

Reply:
Sorry I didn't specify, I meant the spacing property

Reply:
fixed